src/backend/shared/constants.ts

```diff
@@ -27,11 +27,25 @@ export const METADATA_KEY_SUFFIX = '/metadata.json';
 export const ASSEMBLY_KEY_SUFFIX = '/assembly.json';
 
 /**
- * The key suffix for assemblies in the provided language. This should NOT be
- * used for the TypeScript/JavaScript assembly however.
+ * The key suffix for a TypeScript doc artifact (root module).
  */
-export function assemblyKeySuffix(lang: TargetLanguage | '*') {
-  return `/assembly-${lang}.json`;
+export const TYPESCRIPT_DOCS_KEY_SUFFIX = docsKeySuffix(DocumentationLanguage.TYPESCRIPT);
+
+/**
+ * The key suffix for a Python doc artifact (root module).
+ */
+export const PYTHON_DOCS_KEY_SUFFIX = docsKeySuffix(DocumentationLanguage.PYTHON);
+
+/**
+ * The key suffix matching any documentation artifact.
+ */
+export const ANY_DOCS_KEY_SUFFIX = docsKeySuffix('*');
```

Comment:
I kinda prefer to have shared prefixes, so they naturally sort next to each other.

```suggestion
export const DOCS_KEY_SUFFIX_TYPESCRIPT = docsKeySuffix(DocumentationLanguage.TYPESCRIPT);

/**
 * The key suffix for a Python doc artifact (root module).
 */
export const DOCS_KEY_SUFFIX_PYTHON = docsKeySuffix(DocumentationLanguage.PYTHON);

/**
 * The key suffix matching any documentation artifact.
 */
export const DOCS_KEY_SUFFIX_ANY = docsKeySuffix('*');
```

Reply:
Done